test/basic_sensors.spec.ts

```
@@ -645,4 +645,48 @@ describe('Basic Sensors', () => {
+    let movingSensorId: string;
+
+    beforeEach(() => {
+      // Only test service creation for first test case and reuse harness afterwards
+      if (deviceExposes.length === 0 && harness === undefined) {
+        // Load exposes from JSON
+        deviceExposes = loadExposesFromFile('smartthings/im6001-mpp01.json');
+        expect(deviceExposes.length).toBeGreaterThan(0);
+        const newHarness = new ServiceHandlersTestHarness();
+
+        // Check service creation
+        movingSensorId = 'moving_' + hap.Service.MotionSensor.UUID;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)